src/components/Settings/Settings.tsx

```diff
@@ -17,13 +18,14 @@ export const Settings: React.FC<SettingsProps> = ({
   handleAnswerModeChange,
   handleLanguageModeChange,
 }) => {
-  const answerModeText = 'Change answer mode'; // TODO: Translate
-  const languageModeText = 'Change language'; // TODO: Translate
+  const { t } = useTranslation();
+
+  const answerModeText = t('answerModeLabel');
+  const languageModeText = t('languageModeLabel');
```

Comment:
We could move these calls into the component render if that's cleaner. (`<p>{t('...')}</p>`)